include/rmm/spdlog/async.h

```diff
@@ -0,0 +1,93 @@
+// Copyright(c) 2015-present, Gabi Melman & spdlog contributors.
```

Comment:
I think vendors libraries should be in a `thirdparty` directory.

Reply:
Need to check with ops -- I think actually there needs to be a symlinked `thirdparty` directory in the home directory of the repo, too, so that OS dependencies are clear.